projects/swimlane/ngx-ui/src/lib/components/dialog/large-format/components/large-format-dialog-header-action/large-format-dialog-header-action.component.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  HostBinding,
+  Input,
+  Output,
+  ViewEncapsulation
+} from '@angular/core';
+
+@Component({
+  selector: 'ngx-large-format-dialog-header-action',
+  template: `
+    <button
```

Comment:
Should we use ngx-button?

Reply:
What does the `ngx-button` provide us here?